crates/project/src/project_settings.rs

```diff
@@ -141,11 +151,19 @@ pub struct InlineBlameSettings {
     ///
     /// Default: 0
     pub min_column: Option<u32>,
+    /// Whether to show commit summary as part of the inline blame.
+    ///
+    /// Default: false
+    #[serde(default = "false_value")]
+    pub show_commit_summary: bool,
 }
 
 const fn true_value() -> bool {
     true
 }
+const fn false_value() -> bool {
+    true
+}
```

Comment:
I'm new in rust, but is it OK `false_value` function returns a `true` value? 🤔

Reply:
Hehe, good catch. That was fixed in a follow-up PR: https://github.com/zed-industries/zed/pull/19943

Reply:
Also, serde's default for `bool` is `false` so it could've been just `#[serde(default)]` without any extra functions.